### PR TITLE
feat: View all delegates

### PR DESCRIPTION
**Pull request by ncomerci**
image
image

### CODE REVIEW

File: src/components/Home/CommunityEngagement.css [hunk 1]
```diff
@@ -29,6 +29,7 @@
   .CommunityEngagement .CommunityEngagement__Header p {
     font-size: 17px;
     line-height: 28px;
+    width: 60%;
```
Comment (andyesp) [inline src/components/Home/CommunityEngagement.css:32]: Is this looking ok on mobile?

File: src/components/Home/DaoDelegates.tsx [hunk 2]
```diff
@@ -1,29 +1,91 @@
...
+      )}
+      {!loading && (
+        <FullWidthButton onClick={() => handleViewAllDelegatesClick(true)}>
+          {t(isFullList ? 'modal.vp_delegation.details.show_less' : 'page.home.dao_delegates.view_all_delegates')}
+        </FullWidthButton>
+      )}
+      <VotingPowerDelegationModal
+        onClose={handleModalClose}
+        setSelectedCandidate={handleSelectedCandidate}
+        userVP={0}
```
Comment (andyesp) [inline src/components/Home/DaoDelegates.tsx:71]: Why is this 0?

File: src/components/Home/DaoDelegates.tsx [hunk 3]
```diff
@@ -1,29 +1,91 @@
...
       <HomeSectionHeader
         title={t('page.home.dao_delegates.title')}
         description={t('page.home.dao_delegates.description')}
       />
       <Loader active={loading} />
-      {!loading && <DelegatesTable delegates={delegates} userAddress={address} />}
+      {!loading && (
+        <DelegatesTable delegates={delegates} setSelectedCandidate={handleSelectedCandidate} full={isFullList} />
+      )}
+      {!loading && (
```
Comment (andyesp) [inline src/components/Home/DaoDelegates.tsx:63]: You can group both `!loading` conditions.

File: src/components/Home/CommunityEngagement.css (hunk 1 above)
Comment (ncomerci) [reply src/components/Home/CommunityEngagement.css:32]: Yes! this only affects to desktop view

File: src/components/Home/DaoDelegates.tsx (hunk 2 above)
Comment (ncomerci) [reply src/components/Home/DaoDelegates.tsx]: Just for testing, fixed

**Review by andyesp — APPROVED** [review]

**Pull request merged**